### DISCUSSION TITLE
Help installing an older version (missing "include" makes it impossible to work with later versions)

### DISCUSSION THREAD

**Discussion by lvvvcs** [texstudio-org/texstudio#2696] [Q&A]
Hi,
I had to reinstall linux (ubuntu) recently and with it all applications including texstudio. To my dismay, I then saw that the include command on macro scripts has stopped working. This renders the program unusable for me, since I rely heavily on external js scripts which are too long to add to the config file. I was trying to install a previous version, but it seems impossible without having something called qt4 installed, something that seems impossible at the moment. What can I do?
Best

**Comment by lvvvcs** [comment]
I found some intermediate versions (3.x) that still support include but use qt5. Still, no success.  I get the message
Unknown module(s) in QT: script uitools 
and none of the fixes I can find online works.

**Comment by sunderme** [comment]
https://build.opensuse.org/package/show/home:jsundermeyer/texstudio-3.1.2
or compile locally, see wiki
